ada_feeding_actionselect/ada_feeding_actionselect/policies/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+This module contains policies for selecting actions
+based on context (visual and posthoc).
+"""
+
+# Generic Policy
+from .constant_policy import Policy
```

Comment:
Again, it seems unintuitive to me that the abstract class policy is in the `constant_policy.py` file. Perhaps rename that file to `base_policies.py`? (Which would also align well with the suggested change of `base_adapters.py`)